### ISSUE TITLE
Schiene fails to convert price to float

### ISSUE THREAD

**Issue by snizzleorg** [kennell/schiene#6]
```
File "/usr/local/lib/python3.4/site-packages/schiene/schiene.py", line 102, in connections
    return parse_connections(rsp.text)
  File "/usr/local/lib/python3.4/site-packages/schiene/schiene.py", line 33, in parse_connections
    data['price'] = float(data['price'])
ValueError: could not convert string to float: 'ab\xa02.70 EUR'
```

**Comment by kennell** [comment]
Seems there was a change to the Bahn.de code. Should be a easy fix but i'm very busy with other stuff right now. If you want to tackle this yourself, feel free to open a pull request, otherwise it will be a few days until i get to it.

**Comment by derhuerst** [comment]
Check out [db-hafas](https://github.com/derhuerst/db-hafas)!

**Comment by fabaff** [comment]
It still seems to work for certain connections:

```python
>>> import schiene
>>> s = schiene.Schiene()
>>> for key, value in s.connections('Pforzheim', 'Stuttgart')[0].items():
...     print(key, value)
... 

transfers 0
details http://mobile.bahn.de/bin/mobil/query.exe/dox?ld=150100&n=1&i=3t.018078100.1473106151&rt=1&use_realtime_filter=1&co=C0-0&vca&HWAI=CONNECTION$C0-0!details=opened!&
price 12.3
ontime True
arrival 23:25
time 0:54
departure 22:31
products ['RE']
```

**Comment by kennell** [comment]
Fixed and released on PyPI as version 0.18
